problemsets-mat223/modules/module10.tex

```diff
@@ -35,12 +35,21 @@
 our methods will be ad hoc.
 
 \begin{example}
-	Let $\mathcal U:\R^2\to\R^2$ be the transformation induced by $M=\matc{\sqrt{2}/2&-\sqrt{2}/2\\0&0}$,
+	Let $\mathcal U:\R^2\to\R^2$ be the transformation given by $M=\matc{\sqrt{2}/2&-\sqrt{2}/2\\0&0}$,
 	let $\mathcal R:\R^2\to\R^2$ be rotation counter clockwise by $45^\circ$, and let $\mathcal P:\R^2\to\R^2$
 	be projection onto the $x$-axis.
 	Write $\mathcal U$ as the composition (in some order) of $\mathcal R$ and $\mathcal P$.
 
-	XXX Finish
+	For vector $\xhat$ and $\yhat$, Notice that
+	\[
+	    \mathcal U(\xhat)=\matc{\frac{\sqrt{2}}{2}\\0}\\\qquad\text{and}\qquad \mathcal U(\yhat)=\matc{-\frac{\sqrt{2}}{2}\\0},
+	\]
+	and
```

Comment:
`and` -> `We will compute $\mathcal R\circ \mathcal P$ and $\mathcal P\circ\mathcal R$ applied to $\xhat$ and $\yhat$ and see which one matches.